app/collector/main.py

```diff
@@ -0,0 +1,19 @@
+"""This module provides app initialization."""
+
+from aiohttp.web import Application, run_app
+
+from collector.views import routes
+
+
+async def init_app():
+    """Initialize application with routes"""
+    app = Application()
+    app.add_routes(routes)
+    return app
+
+
+# todo: move this to app package into manager
+def main():
+    """Run app on certain host and port"""
+    app = init_app()
+    run_app(app, port=8000)
```

Comment:
It would be useful to add the ability to set custom port and add ability to run in debug mode. And as you mention in the docstring we should add host configuration too.

Reply:
It was planned to add with manager, but it was rejected